package.json

```diff
@@ -223,7 +223,6 @@
         "@types/react-resizable": "^3.0.4",
         "@types/react-syntax-highlighter": "^15.5.7",
         "@types/react-textfit": "^1.1.0",
-        "@types/react-transition-group": "^4.4.4",
```

Comment:
fly-by removal - we also have the types for 4.4.5 _and_ we have 4.4.5 of the actual library installed